src/EPR.SubsidiaryBulkUpload.Application/Services/ISubsidiaryService.cs

```diff
@@ -16,4 +16,6 @@ public interface ISubsidiaryService
     Task<bool> GetSubsidiaryRelationshipAsync(int parentOrganisationId, int subsidiaryOrganisationId);
 
     Task<UserOrganisation> GetSystemUserAndOrganisation();
+
+    Task<OrganisationResponseModel?> GetCompanyByLocalDBCompanyName(string companyName);
```

Comment:
No need to say "LocalDB" - GetCompanyByCompanyName would be better

Reply:
changed